src/dialog/improveAbilityDialog.ts

```diff
@@ -0,0 +1,594 @@
+import { Rune } from "../actors/item-specific/rune";
+import { RqgActor } from "../actors/rqgActor";
+import { Characteristic } from "../data-model/actor-data/characteristics";
+import { ItemTypeEnum } from "../data-model/item-data/itemTypes";
+import { RuneDataProperties } from "../data-model/item-data/runeData";
+import { Ability, IAbility } from "../data-model/shared/ability";
+import { RqgItem } from "../items/rqgItem";
+import { assertItemType, getGame } from "../system/util";
+
+export class ImproveAbilityDialog {
+  //**Shows a dialog for improving a Passion, Rune, or Skill */
```

Comment:
I think we can skip the class and export the functions directly. I'm not consistent in the system, and when I started I leaned against making classes for things (coming from Java / Scala) but I now think that it doesn't add anything - at least not if the class doesn't store something as well. It's definitely a matter of preference, but I would say that we should not use "function holding classes".

Reply:
done